mkdocs.yml

```
@@ -33,7 +33,8 @@ nav:
     - glance_adoption.md
     - other_services_adoption.md
   - Ceph:
-    - ceph.md
+    - ceph_rbd.md
```

@fao89 @jistr I run this locally (`mkdocs serve`) and I see the two sections properly rendered with their title.

---

FYI: you can also rename it here:
```suggestion
    - Ceph new name: ceph_rbd.md
```